### PR TITLE
Fix Segment Response Header Tracking

### PR DESCRIPTION
**Pull request by tomkordic**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -65,7 +65,9 @@ allprojects {
         releaseVersion = 'r2.1.1'
         releaseWebsite = 'https://github.com/google/ExoPlayer'
 
-        muxCoreVersion = "7.0.11"
+//        muxCoreVersion = "7.0.11"
```
Comment (skidder) [inline build.gradle]: Should these comments be removed, or the original text kept in place?

Comment (tomkordic) [reply build.gradle]: I am thinking it should be replaced with new version of MuxCore after it is released, with older versions some tests will fail

Comment (skidder) [reply build.gradle]: Right, so maybe this should be set back to `7.0.11` in the meantime, then?

Comment (tomkordic) [reply build.gradle]: ok

**Review by daytime-em — APPROVED** [review]
Looks great!!

**Pull request merged**